frontend/src/components/LinkButton/LinkButton.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import '../MyNavbar/MyNavbar.css'
+import {Button} from 'reactstrap';
+
+
+
+//Compact button component
+//Set prop 'target' to "_blank" to open the link in a new tab 
+function LinkButton(props) {
+    return(
+        <>
+        <Button style={{ background: 'rgb(176, 216, 201)' }} className="button-background" href = {props.url} target={props.target}>
```

Comment:
What do you think about moving the style into a stylesheet?

Reply:
Done!

Reply:
I needed to add the !important tag to the CSS file so that it would work.